jkube-kit/build/service/docker/src/test/java/org/eclipse/jkube/kit/build/service/docker/helper/ConfigHelperTest.java

```diff
@@ -30,16 +31,27 @@
 import java.util.Properties;
 
 import static org.assertj.core.api.AssertionsForInterfaceTypes.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.RETURNS_DEEP_STUBS;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
 
 public class ConfigHelperTest {
-  @Mocked
   private ImageConfigResolver imageConfigResolver;
-
-  @Mocked
   private KitLogger logger;
-
-  @Mocked
   private JavaProject javaProject;
+  private JKubeConfiguration jKubeConfiguration;
+
+  @Before
+  public void setUp() {
+    imageConfigResolver = mock(ImageConfigResolver.class, RETURNS_DEEP_STUBS);
+    logger = mock(KitLogger.class, RETURNS_DEEP_STUBS);
```

Comment:
If we are not verifying any of the logged messages, `SilentLogger` is a good option for tests instead of using a mock.

Reply:
Do you have some examples of using it? I'm not able to find any references to `SilentLogger` in this repository